bin/gaffer

```diff
@@ -90,7 +90,10 @@ export GAFFER_STARTUP_PATHS=~/gaffer/startup:${GAFFER_STARTUP_PATHS:-}${GAFFER_S
 
 export GAFFERUI_IMAGE_PATHS=$GAFFER_ROOT/graphics${GAFFERUI_IMAGE_PATHS:+:}${GAFFERUI_IMAGE_PATHS:-}
 
-export OSLHOME=$GAFFER_ROOT
+if [[ -z $OSLHOME ]] ; then
```

Comment:
If Gaffer has an internal OSL build, it should really be used in preference to any external build to avoid unwanted clashes. Can we replace this with `[[ -e $GAFFER_ROOT/bin/python/oslc ]]`, following the same logic as we use for setting up `PYTHONHOME` below?